cdn/dev/keyboard-search/dedicated-landing-pages.js

```diff
@@ -0,0 +1,66 @@
+//
+// Meta search results for dedicated landing pages
+//
+
+const dedicatedLandingPages = [
+  {
+    id: "/amharic",
+    terms: ['amharic', 'ge\'ez', 'geez', 'ethiopic'],
+    description: "Amharic keyboards recommended by the Keyman team",
+    name: "Amharic Keyboards Home Page"
+  },
+  {
+    id: "/burmese",
+    terms: ['burmese', 'myanmar'],
+    description: "Burmese (Myanmar) keyboards recommended by the Keyman team",
+    name: "Burmese Keyboards Home Page"
+  },
+  {
+    id: "/cameroon",
+    terms: ['cameroon'],
+    description: "Cameroon keyboards recommended by the Keyman team",
+    name: "Cameroon Keyboards Home Page"
+  },
+  {
+    id: "/greek",
+    terms: ['greek', 'ancient greek', 'biblical greek', 'attic greek', 'classical greek', 'polytonic', 'polytonic greek'],
+    description: "Biblical and Classical Greek keyboards recommended by the Keyman team",
+    name: "Classical Greek Keyboards Home Page"
+  },
+  {
+    id: "/ipa",
+    terms: ['ipa', 'phonetic', 'phonetic alphabet', 'international phonetic alphabet', 'transcription'],
+    description: "IPA (International Phonetic Alphabet) keyboards recommended by the Keyman team",
+    name: "IPA Keyboards Home Page"
+  },
+  {
+    id: "/sinhala",
+    terms: ['sinhala', 'sinhalese'],
+    description: "Sinhala keyboards recommended by the Keyman team",
+    name: "Sinhala Keyboards Home Page"
+  },
+  {
+    id: "/tamil",
+    terms: ['tamil'],
+    description: "Tamil keyboards recommended by the Keyman team",
+    name: "Tamil Keyboards Home Page"
+  },
+  {
+    id: "/tibetan",
+    terms: ['tibetan', 'dzongkha'],
+    description: "Tibetan and Dzongkha keyboards recommended by the Keyman team",
+    name: "Tibetan Keyboards Home Page"
+  },
+  {
+    id: "/tigrigna",
+    terms: ['tigrigna', 'tigrinya'],
```

Comment:
Do we want a separate term `eritrea` which goes to /tigrigna/eritrean ?

Reply:
Perhaps in the future